### ISSUE TITLE
Format of genome_size input?

### ISSUE THREAD

**Issue by evautumn** [emyounglab/prymetime#36]
Hello, 
What is the format of the genome_size input? Is it in bp, Mbp, or Gbp? How does this feature affect the program? If the program predicts a genome size of 11 Mb with the nanopore and illumina files, and my genome-size input is 14 Mb, does it change anything? 
If I write `-genome-size 10845397 ` (as in 10845397 bp), is this acceptable? 
Thanks!

**Comment by sjtrauber** [comment]
Yes, that is acceptable. Adding genome size to Prymetime is for Flye to run more effectively, but it is completely optional for Flye with the 2.8 update. For Prymetime, without genome-size specified, it defaults to set genome-size to 12100000bp.
